src/components/block/index.ts

```diff
@@ -296,18 +227,17 @@ export default class Block extends EventsDispatcher<BlockEvents> {
     api,
     readOnly,
     tunesData,
-  }: BlockConstructorOptions) {
+  }: BlockConstructorOptions, eventBus?: EventsDispatcher<EditorEventMap>) {
     super();
 
     this.name = tool.name;
     this.id = id;
     this.settings = tool.settings;
     this.config = tool.settings.config || {};
     this.api = api;
+    this.editorEventBus = eventBus || null;
```

Comment:
Maybe we could make api instance of EventDispatcher, so we dont need to pass two parameters. Or maybe have a property of the api object which is EventDispatcher instance

Reply:
do you want to expose editor internal event bus to users? I think, it's better to do design it first and implement via a separate feature